README.md

```diff
@@ -92,10 +92,54 @@ dashbook
       );
     });
 
-    return Text('Use actions to show the Dialog');
+    return SizedBox();
   });
 ```
 
+## Example information
+
+More than often, an example may not be intuitive enough and the user may be lost without some instruction on how to interact with it. To mitigate that, text information can be linked to an example to serve as a guide, or really to show any relevant information.
```

Comment:
-More than
just start with "Often" and no comma
(The saying you were looking for is "More often than not", but I think it is easier to read with just often)

```suggestion
Often an example may not be intuitive enough and the user may be lost without some instruction on how to interact with it. To mitigate that, text information can be linked to an example to serve as a guide, or to show any other relevant information.
```